pkg/resourcegenerator/job/job.go

```diff
@@ -127,6 +132,7 @@ func getJobSpec(skipJob *skiperatorv1alpha1.SKIPJob, selector *metav1.LabelSelec
 	// it's not a default label, maybe it could be?
 	// used for selecting workloads by netpols, grafana etc
 	jobSpec.Template.Labels["app"] = skipJob.KindPostFixedName()
+	jobSpec.Template.Labels["app.kubernetes.io/version"] = resourceutils.GetImageVersion(skipJob.Spec.Container.Image)
```

Comment:
a lot of duplicate code here. 
Could be good to move it into a job_utils.go file in this package or into resourceutils so we have a single function for setting job workload labels

Reply:
or just create a function at the end of this file

Reply:
Fixed!